handbook/people-ops/index.md

```diff
@@ -9,7 +9,7 @@
 - [Spending company money](spending-company-money.md)
 - [Glossary](from-graphbook/glossary.md)
 - [Vacation/FTO](fhttps://github.com/sourcegraph/Graphbook/blob/master/Holidays,%20FTO,%20and%20leaves%20of%20absence/Vacation%20and%20FTO.md)
```

Comment:
Status checks are failing because this link is broken

```suggestion
- [Vacation/FTO](https://github.com/sourcegraph/Graphbook/blob/master/Holidays,%20FTO,%20and%20leaves%20of%20absence/Vacation%20and%20FTO.md)
```

Reply:
Ah :) thanks!